modules/client-configuration/pages/contact-methods-migrate-traditional.adoc

```diff
@@ -10,26 +10,29 @@ This could be done by:
 * running the bootstrap script to register the client, with the aforementioned reactivation key.
 
 
+
 == Generate Reactivation Key
 
-A reactivation key can be used to re-register the client and regain all {productname} settings. 
-For more information on how to create reactivation key, see xref:client-configuration:activation-keys.adoc#reactivation_keys[Reactivation Keys].
+A reactivation key can be used to re-register the client and regain all {productname} settings.
+For more information on how to create reactivation key, see xref:client-configuration:activation-keys.adoc#reactivation_keys[].
 // For more information on how to create reactivation key, see xref:client-configuration:activation-keys.adoc[].
 
 
+
 == Create Bootstrap Script
-.Procedure: Creating Bootstrap Script
-A bootstrap script can be created through the {webui} or from the command line. 
+
+A bootstrap script can be created with the {webui} or from the command line.
 For more information about creating bootstrap script, see xref:client-configuration:registration-bootstrap.adoc[].
-[NOTE]
-====
-Use  the activation key created before to generate the bootstrap script.
-====
 
-. Once the bootstrap script is generated, it's stored on the server in the [path]``/srv/www/htdocs/pub/bootstrap`` directory.
-  Alternatively, you can access the bootstrap script over HTTPS.
-  Replace [literal]``<example.com>`` with the host name of your {productname} Server:
-+
+// FiXME: Where exactly did we create this activation key?
+//        Do we actually need this hint?
```

Comment:
Yes, it's really important to use the activation key when creating the bootstrap script for this or it just won't work.

Reply:
ok